modules/ding_seo/ding_seo.features.metatag.inc

```diff
@@ -0,0 +1,145 @@
+      'schema_event.isAccessibleForFree' => array(
+        'value' => '[node:ding_seo_node_tokens:event_free]',
+      ),
+      'schema_event.startDate' => array(
+        'value' => '[node:ding_seo_node_tokens:event_startdate]',
+      ),
+      'schema_event.endDate' => array(
+        'value' => '[node:ding_seo_node_tokens:event_enddate]',
+      ),
+      'schema_event.location' => array(
+        'value' => 'a:5:{s:5:"@type";s:5:"Place";s:4:"name";s:41:"[node:ding_seo_node_tokens:location_name]";s:3:"url";s:40:"[node:ding_seo_node_tokens:location_url]";s:7:"address";a:6:{s:5:"@type";s:13:"PostalAddress";s:13:"streetAddress";s:41:"[node:ding_seo_node_tokens:streetAddress]";s:15:"addressLocality";s:43:"[node:ding_seo_node_tokens:addressLocality]";s:13:"addressRegion";s:41:"[node:ding_seo_node_tokens:addressRegion]";s:10:"postalCode";s:38:"[node:ding_seo_node_tokens:postalCode]";s:14:"addressCountry";s:42:"[node:ding_seo_node_tokens:addressCountry]";}s:3:"geo";a:3:{s:5:"@type";s:14:"GeoCoordinates";s:8:"latitude";s:36:"[node:ding_seo_node_tokens:latitude]";s:9:"longitude";s:37:"[node:ding_seo_node_tokens:longitude]";}}',
+      ),
+      'schema_event.url' => array(
+        'value' => '[node:url]',
+      ),
+    ),
+  );
+
+  // Exported Metatag config instance: node:ding_library.
+  $config['node:ding_library'] = array(
+    'instance' => 'node:ding_library',
+    'disabled' => FALSE,
+    'config' => array(
+      'schema_organization.@type' => array(
+        'value' => 'Library',
+      ),
+      'schema_organization.@id' => array(
+        'value' => '[site:url]',
+      ),
+      'schema_organization.name' => array(
+        'value' => '[node:title]',
+      ),
+      'schema_organization.address' => array(
+        'value' => 'a:5:{s:5:"@type";s:13:"PostalAddress";s:13:"streetAddress";s:41:"[node:ding_seo_node_tokens:streetAddress]";s:15:"addressLocality";s:30:"[dingseotoken:addressLocality]";s:10:"postalCode";s:38:"[node:ding_seo_node_tokens:postalCode]";s:14:"addressCountry";s:42:"[node:ding_seo_node_tokens:addressCountry]";}',
+      ),
+      'schema_organization.geo' => array(
+        'value' => 'a:3:{s:5:"@type";s:14:"GeoCoordinates";s:8:"latitude";s:36:"[node:ding_seo_node_tokens:latitude]";s:9:"longitude";s:37:"[node:ding_seo_node_tokens:longitude]";}',
+      ),
+      'schema_organization.telephone' => array(
+        'value' => '[node:field_ding_library_phone_number]',
+      ),
+      'schema_organization.url' => array(
+        'value' => '[site:url]',
+      ),
+      'schema_organization.description' => array(
+        'value' => '[node:field_ding_library_lead]',
+      ),
+      'schema_organization.image' => array(
+        'value' => 'a:4:{s:5:"@type";s:11:"ImageObject";s:3:"url";s:61:"[node:field_ding_library_title_image:ding_primary_large:path]";s:5:"width";s:62:"[node:field_ding_library_title_image:ding_primary_large:width]";s:6:"height";s:63:"[node:field_ding_library_title_image:ding_primary_large:height]";}',
```

Comment:
We have used the same image styles as is used for the title image in the content. I think this makes most sense, since editors will use an image that works in that format.